rio/src/RIO/List/Partial.hs

```diff
@@ -17,8 +17,6 @@ module RIO.List.Partial
   , Data.List.maximumBy
   , Data.List.minimumBy
 
-  -- * Building lists
```

Comment:
There are still changes present in this file versus master, which probably shouldn't be here. Also, I _do_ think it's worth adding a comment that `scanl1` and `scanr1` are here for only historical reasons.